packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -215,6 +215,68 @@ export const APPMESH_ECR_ACCOUNTS: { [region: string]: string } = {
   'us-west-2': '840364872350',
 };
 
+// https://docs.aws.amazon.com/appconfig/latest/userguide/appconfig-integration-lambda-extensions-versions.html
+export const APPCONFIG_LAMBDA_LAYER_ARNS: { [key: string]: any } = {
```

Comment:
Shouldn't [older versions](https://docs.aws.amazon.com/appconfig/latest/userguide/appconfig-integration-lambda-extensions-versions.html#appconfig-integration-lambda-extensions-enabling-older-versions) be added as well?

Reply:
Added